lib/commons/aria/is-accessible-ref.js

```diff
@@ -1,13 +1,24 @@
 /* global aria, axe, dom */
-function findDomNode(node, functor) {
-	if (functor(node)) {
-		return node;
+const idRefsRegex = /^idrefs?$/;
+
+function cacheIdRefs(node, refAttrs) {
+	if (node.hasAttribute) {
+		if (node.nodeName.toUpperCase() === 'LABEL' && node.hasAttribute('for')) {
+			axe._cache.idRefs[node.getAttribute('for')] = 1;
+		}
+
+		refAttrs
+			.filter(attr => node.hasAttribute(attr))
```

Comment:
`hasAttribute(attr)` should be sanitized for space characters.

Reply:
`refAttrs` comes from our lookuptable, I wouldn't think there's a need to sanitize it for whitespace characters.